content/en/platform/corda/4.9/community/release-notes.md

```diff
@@ -17,6 +17,16 @@ tags:
 
 # Corda Community Edition 4.9 release notes
 
+## Corda Community Edition 4.9.5 release notes
+
+Corda Community 4.9.5 is a patch release of Corda Community which includes resolved issues.
+
+### Fixed issues
+
+* Previously, a memory leak in the transaction cache occurred due to weight of in-flight entries being undervalued. Improvements to prevent in-flight entry weights from being undervalued and estimated more correctly means a large decrease in the total size of cached entities. 
```

Comment:
```suggestion
* Previously, a memory leak in the transaction cache occurred due to the weight of in-flight entries being undervalued. Improvements have been made to prevent in-flight entry weights from being undervalued and because they are now estimated more correctly, this results in a large decrease in the total size of cached entities. 
```

Reply:
Done.